### ISSUE TITLE
Some of the new ASGS features don't have Albers Areas

### ISSUE THREAD

**Issue by ashleysommer** [CSIRO-enviro-informatics/asgs-dataset#15]
Namely:
* SignificantUrbanAreas (SUAs)
* IndigenousLocations (ILOCs)
* IndigenousAreas (IAREs)
* IndigenousRegions (IREGs)
* UrbanCentreAndLocailities (UCLs)
* SectionOfStateRegions (SOSRs)
* SectionOfStates (SOSs)

Each do no have an Albers Area property provided by the ASGS WFS endpoint.
Without an albers area, area-based apportionment tool which uses equal-area projections for its gemoetries (ie, Excellerator) will not work. We might have to compute these ourselves using PostGIS and store them somewhere (in the Geometry service?)

**Comment by ashleysommer** [comment]
Two of the new feature types _do_ have Albers Area:
* GreaterCapitalCityStatisticalAreas (GCCSAs)
* RemotenessAreas (RAs)

So we can map the provided albers areas to `hasAreaM2` directly same as done with Mesblocks, SA1->SA4 etc.

**Comment by jyucsiro** [comment]
@ashleysommer that's interesting. on missing areas - yes, in that case we'd need to compute these in PostGIS. We can use the Geometry service's instance of PostGIS for that. @benjaminleighton was suggesting we consolidate the PostGIS operations for producing linksets to that also. We could also output those areas as a static file and host on S3?

**Comment by ashleysommer** [comment]
@jyucsiro 
I've just realized these Albers areas can be calculated a different way.

We have Albers areas for SA1s and SA2s. ILOCs and UCLs are made up of groups of SA1s. SUAs are made of groups of SA2s.

So aggregating Albers areas of the building blocks _should_ give us the Albers of the new features.

**Comment by jyucsiro** [comment]
cool - yes that's an alternative too

**Comment by jyucsiro** [comment]
Full list that don't have albers area:

SUA, e.g. http://linked.data.gov.au/dataset/asgs2016/significanturbanarea/1018
ILOC, e.g. http://linked.data.gov.au/dataset/asgs2016/indigenouslocation/50800204
IREG, e.g. http://linked.data.gov.au/dataset/asgs2016/indigenousregion/501
UCL, e.g. http://linked.data.gov.au/dataset/asgs2016/urbancentreandlocality/112001
SOSR, e.g. http://linked.data.gov.au/dataset/asgs2016/sectionofstaterange/203
SOS, e.g. http://linked.data.gov.au/dataset/asgs2016/sectionofstate/50
CED, http://linked.data.gov.au/dataset/asgs2016/commonwealthelectoraldivision/237
SSC, http://linked.data.gov.au/dataset/asgs2016/statesuburb/60527
NRMR, http://linked.data.gov.au/dataset/asgs2016/naturalresourcemanagementregion/401

Without these, it isn't currently possible to calculate proportion overlaps/contains between 2 features.

Affects https://github.com/CSIRO-enviro-informatics/loci-integration-api and applications dependant on it

**Comment by jyucsiro** [comment]
@ashleysommer If the ABS endpoints were to include them, would this be an easy fix?

**Comment by ashleysommer** [comment]
Yes, if ABS wfs endpoints added these, it is an easy fix. But I've checked not only their wfs, but their geoserver's json query endpoint too, and that data does not exist.
